src/PhpSpec/Console/Application.php

```diff
@@ -147,6 +147,13 @@ protected function loadConfigurationFile(InputInterface $input, ServiceContainer
 
                     $extension->load($container);
                 }
+            } elseif ('matchers' === $key && is_array($val)) {
+                foreach ($val as $class) {
+                    $id = strtolower(str_replace('Matcher', '', substr($class, strrpos($class, '\\') + 1)));
```

Comment:
Classes don't have to have namespaces, so you need to handle the case where `strpos` here returns `false`.

I don't really see the benefit here of assuming that the string `'Matcher'` will be in the output - it won't hurt if it's in the `$id`